ocs/ocp.py

```diff
@@ -65,7 +65,13 @@ def exec_oc_cmd(self, command):
 
         oc_cmd += command
         out = run_cmd(cmd=oc_cmd)
-        return munchify(yaml.safe_load(out))
+
+        out = yaml.safe_load(out)
+        # In some cases, run_cmd() will return an empty string or in the case
+        # of executing ceph commands, it could return an empty list
+        if out != list():
+            return munchify(out)
+        return out
```

Comment:
Is this actually needed? I see that for both empty string and empty and plain lists, assumption `l == munch.munchify(l)` holds.

Reply:
@mbukatov  Correct, this is not needed actually